kanidm_tui/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "kanidm_tui"
+version = "0.1.0"
+authors = ["George Rose (Geo-7) <george@1w1g.com>"]
+rust-version = "1.59"
+edition = "2021"
+license = "MPL-2.0"
+description = "Kanidm Terminal UI"
+documentation = "https://docs.rs/kanidm_tools/latest/kanidm_tui/"
+homepage = "https://github.com/kanidm/kanidm/"
+repository = "https://github.com/kanidm/kanidm/"
+ 
+
+[dependencies]
+kanidm_client = {version = "1.1.0-alpha.4" }
```

Comment:
As yaleman mentioned, since it's a relative import you should use path = " " instead.

Reply:
It is done.